README.md

```diff
@@ -55,7 +55,50 @@ For most existing Dojo 1 Dijits, the TypeScript typings can be found at [dojo/ty
 
 ### ReduxInjector
 
-*Coming Soon*
+`ReduxInjector` can be used to bind a redux store and Dojo 2 widgets using the `registry`.
+
+An injector can be defined in the registry, which is then registered with the `Router`, and provided to the `Projector` as one of its properties. This is demonstrated in the example below.
+
+```typescript
+import global from '@dojo/shim/global';
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import { registerRouterInjector } from '@dojo/routing/RouterInjector';
+import { ReduxInjector } from '@dojo/interop/redux/ReduxInjector';
+import { Registry } from '@dojo/widget-core/Registry';
+
+import { TodoAppContainer } from './containers/TodoAppContainer';
+import { createStore } from 'redux';
+import { todoReducer } from './reducers';
+
+const defaultState = {
+	todos: [],
+	currentTodo: '',
+	activeCount: 0,
+	completedCount: 0
+};
+
+const registry = new Registry();
+const store = createStore(todoReducer, defaultState, global.__REDUX_DEVTOOLS_EXTENSION__ && global.__REDUX_DEVTOOLS_EXTENSION__());
```

Comment:
might want to call out that you this initialises the redux dev tools extension (and maybe a link to it?).

Reply:
do we need this in the readme?

Reply:
Nope probably not to be honest.

Reply:
I removed the redux dev tools extension from the example code and removed the explanation about it.